packages/core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -24,14 +24,22 @@ export const DatePicker: React.FC<Props> & WithStyle = React.memo(
                 popoverPlacement,
                 minSelectableDate,
                 maxSelectableDate,
+                mobile,
```

Comment:
I think you've probably made `mobile` a prop instead of relying on the window size so that the application using this library can make their own decision about how it detects mobile users, but it seems like these changes would be good for any small screen. I've begun adding some responsive utils which could be used if it seems like that direction makes sense.

Reply:
I agree with jared

Reply:
I've spoken to Travis about this before and my initial understanding was a per device (non-window size) approach. I ultimately do prefer just using pure css for this so I'll reach out about how much flexibility we have.

Reply:
from @travisrozich 
`Should technically be dependent on device - because you could be at a mobile breakpoint, but looking at it on chrome on a laptop. So I think since we're pulling the native / system's datepicker, this should be based on device (or Operating system, if that's detectable)`